src/Symfony/Bundle/FrameworkBundle/Tests/Command/WorkflowDumpCommandTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Tests\Command;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Bundle\FrameworkBundle\Command\WorkflowDumpCommand;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Tester\CommandCompletionTester;
+
+class WorkflowDumpCommandTest extends TestCase
+{
+    /**
+     * @dataProvider provideCompletionSuggestions
+     */
+    public function testComplete(array $input, array $expectedSuggestions)
+    {
+        $application = new Application();
```

Comment:
Hum just missing the exclude case in case <v5.3
see https://github.com/symfony/symfony/blob/df08087f9809646aed35af8a5a4db98ecba3e558/src/Symfony/Component/PasswordHasher/Tests/Command/UserPasswordHashCommandTest.php#L296

Reply:
Not needed as the test is part of framework bundle.

Reply:
According PR https://github.com/symfony/symfony/pull/43864 it is not needed.

Reply:
ho got it! thanks, +1 for moving the command then :)